docs/quickstart/send-report-ui.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: "Send observability report"
+---
+
+import InstallCLI from "../snippets/install-cli.mdx";
+import InstallDbtPackage from "../snippets/install-dbt-package.mdx";
+
+Distribute the report by sending it to external platforms using the command `edr monitor send-report`.  
+Each specified platform in the command will be sent a report.  
+All arguments can be passed either as CLI arguments or in the `config.yml` file.
+
+## Slack
+
+After you [setup a Slack app and token](../integrations/slack#slack-integration-setup) you can run the command:
+
+```shell
+edr monitor send-report --slack-token <SLACK_TOKEN> --slack-channel-name <CHANNEL_NAME>
+```
+
+A report will be sent to your Slack channel.
+
```

Comment:
Maybe refer here to the multiple channels and subscribers config we have? 
We want to promote it.

Reply:
Wouldn't it be confusing to the user to see other configuration values given that it's unrelated to the report?